src/components/shared/Tabs.jsx

```diff
@@ -0,0 +1,59 @@
+import classnames from 'classnames'
+import React, { useState } from 'react'
+
+import classNames from './Tabs.scss'
+
+export const Tabs = ({ children, initialTab = 0, ...props }) => {
+  const [tab, setTab] = useState(children[initialTab])
+
+  return (
+    <section className={classNames.root}>
+      <TabSelector
+        setTab={setTab}
+        tab={tab}
+        tabs={children}
+      />
+      <TabContent tab={tab} />
+    </section>
+  )
+}
+
+const TabSelector = ({ setTab, tab, tabs, ...props }) => {
+  const onClick = tab => () => {
+    setTab(tab)
+  }
+
+  return (
+    <section className={classNames.selector}>
+      <div className={classNames.selectorRail} />
+      {tabs.map(t => (
+        <TabLabel
+          active={t.props.label === tab.props.label}
+          label={t.props.label}
+          onClick={onClick}
+          tab={t}
+        />
+      ))}
+      <div className={classNames.selectorRail} />
+    </section>
+  )
+}
+
+const TabLabel = ({ active, label, onClick, tab }) => (
+  <div
+    className={classnames(classNames.label, { [classNames.labelActive]: active })}
+    onClick={onClick(tab)}
```

Comment:
I think we could simplify this a little bit by not even passing the `tab` - just calling `onClick`. Whoever instantiated this `TabLabel` already has this `tab`, so they should know what to do.